crates/biome_fs/src/dir.rs

```diff
@@ -0,0 +1,15 @@
+use directories::ProjectDirs;
+use std::{env, fs, path::PathBuf};
+
+pub fn ensure_data_dir() -> PathBuf {
+    let project_data_dir = if let Some(proj_dirs) = ProjectDirs::from("dev", "biomejs", "biome") {
+        // Linux: /home/alice/.local/share/biome
+        // Win: C:\Users\Alice\AppData\Roaming\biomejs\biome\Data
+        // Mac: /Users/Alice/Library/Application Support/dev.biomejs.biome
```

Comment:
Out of curiosity, what does the `dev.` prefix mean here?

Reply:
The domain postfix; usually `com`, but in this case `dev` from `biomejs.dev`.

Reply:
Oh right, sorry it makes sense now.